contrib/redhat/collectd.spec

```diff
@@ -16,7 +16,7 @@
 #
 # - fetch the desired collectd release file from https://collectd.org/files/
 #   and save it in your ~/rpmbuild/SOURCES/ directory (or build your own out of
-#   the git repository: ./build.sh && ./configure && make dist-bzip2)
+#   the git repository: ./build.sh && ./configure && make-dist-bz2)
```

Comment:
Please rebase on *master* since this reverts #2459.

Reply:
error in jenkins comes from https://github.com/collectd/collectd/commit/ea341582f90a9a7ee4b5525551edc2fdb4ec1226
Build #988 (Sep 27, 2017 8:25:07 AM)